src/core.ts

```diff
@@ -2,7 +2,7 @@ const patterns = {
   route: [/\/src\/pages\/|\.(jsx|tsx)$/g, ''],
   splat: [/\[\.{3}.+\]/, '*'],
   param: [/\[([^\]]+)\]/g, ':$1'],
-  slash: [/index|\./g, '/'],
+  slash: [/^index|\./g, '/'],
```

Comment:
Thanks for this fix! Could we also match the end of the `index` word beside matching the beginning as you added?

```suggestion
  slash: [/^index$|\./g, '/'],
```